scorecard/profile_data.py

```diff
@@ -1492,7 +1494,7 @@ def calendar_to_financial(year, month):
 
 
 def quarter_idx(month):
-    return ((month - 1) / 3) + 1
+    return ((month - 1) // 3) + 1
```

Comment:
I'm not sure floor is what we wanted here, but maybe! In my head this was Python2 5/3 = 2 or Python3 round(5/3) = 2 but apparently it was Python2 5/3 = 1 so perhaps this is fine...

http://plasmasturm.org/log/6debug/

Reply:
This looks fine to me. It calculates the quarter from a 1-based month count. the `//` does integer division.